topics/epigenetics/tutorials/cut_and_run/tutorial.md

```diff
@@ -0,0 +1,595 @@
+---
+layout: tutorial_hands_on
+
+title: CUT & RUN data analysis
+zenodo_link: https://zenodo.org/record/3862793
+questions:
+- Which binding motif has the transcription factor GATA1?
+- What kind of quality control do I have to do?
+- How to analyse and visualise CUT & RUN data?
+objectives:
+- Apply appropriate analysis and quality control steps for CUT & RUN
+- Apply an enrichment analysis and a robust peak detection
+- Find protein binding motifs
+time_estimation: 3h
+key_points:
+- CUT & RUN can be used to identify binding regions for
+- Several filters are applied to the reads, such as removing those mapped to mitochondria
+- Fragment distribution can help determine whether an CUT & RUN experiment has worked well
+- A robust peak detection removes noise from the data
+contributors:
+- heylf
+
+---
+
+# Introduction
+{:.no_toc}
+
+
+In many organism transcription factors (TF) play an important tole in the regulation of the gene expression. In human, we have up to 2,800 proteins and more than 1,600 are TF ([list of transcription factors](https://en.wikipedia.org/wiki/List_of_human_transcription_factors)), although the number might change over timer. Investigating the role of TFs, such as [GATA1](https://en.wikipedia.org/wiki/GATA1), is a very important task to understand the regulatory mechanisms in the cell and thus ascertain the source of a disease, such as [myelofibrosis](https://en.wikipedia.org/wiki/Primary_myelofibrosis) a type of blood cancer.  
+
+![CUT_and_RUN](../../images/cut_and_run/cut_and_run.png "Zhu et al. 2019 Genome Biology")
+
+**C**leavage **U**nder **T**argets and **R**elease **U**sing **N**uclease (CUT&RUN) {% cite Skene2017 %} became the new and advanced method to analyse DNA-associated proteins. CUT&RUN uses an antibody just as [ChIP-Seq](https://en.wikipedia.org/wiki/ChIP_sequencing) to select the protein of interest (POI). The big difference, CUT&RUN couples the antibody with a protein A-micrococcal nuclease (pA-MNase), which you can see in **Figure 1**. The [enzyme](https://en.wikipedia.org/wiki/Micrococcal_nuclease) is an endo-exonuclease that cleaves and shortens the bound DNA of the selected POI *in-situ*. CUT&RUN allows to fragment the DNA in intact cells and thus allows to study protein-DNA interactions in a more natural state. The added pA-MNase thus creates shorted fragments that lead to a higher resolution for the mapping in comparison to your standard ChIP-Seq protocol. CUT&RUN follows four fundamental steps: (1) fixate and make the nuceli or cells permeable, (2) add selective antibody of the POI, (3) add and activate pA-MNas, (4) release DNA complex and collect the DNA from the supernatant. Afterwards, the DNA can be PCR amplified and prepared for sequencing.
```

Comment:
```suggestion
{CUT&RUN} {% cite Skene2017 %} became the new and advanced method to analyse DNA-associated proteins. CUT&RUN uses an antibody just as [ChIP-Seq](https://en.wikipedia.org/wiki/ChIP_sequencing) to select the protein of interest (POI). The big difference, CUT&RUN couples the antibody with a protein A-micrococcal nuclease (pA-MNase), which you can see in **Figure 1**. The [enzyme](https://en.wikipedia.org/wiki/Micrococcal_nuclease) is an endo-exonuclease that cleaves and shortens the bound DNA of the selected POI *in-situ*. CUT&RUN allows to fragment the DNA in intact cells and thus allows to study protein-DNA interactions in a more natural state. The added pA-MNase thus creates shorted fragments that lead to a higher resolution for the mapping in comparison to your standard ChIP-Seq protocol. CUT&RUN follows four fundamental steps: (1) fixate and make the nuceli or cells permeable, (2) add selective antibody of the POI, (3) add and activate pA-MNas, (4) release DNA complex and collect the DNA from the supernatant. Afterwards, the DNA can be PCR amplified and prepared for sequencing.
```

maybe a good place to use abbreviations instead? It'll render a bit more nicely for screenreaders. https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#abbreviations

just add

```
abbreviations:
  CUT&RUN: Cleavage Under Targets and Release Using Nuclease
```

to the metadata of this tutorial

Reply:
Awesome. That totally makes sense.